examples/with-compiled-css/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
+}
```

Comment:
the additional plugin is required, for me it doesn't work without it, so `.babelrc` should look like:

```
{
  "presets": ["next/babel"],
  "plugins": ["@babel/plugin-syntax-jsx"]
}
```